code/game/machinery/kitchen/stove.dm

```diff
@@ -0,0 +1,256 @@
+/obj/machinery/stove
+	name = "electric stove"
+	desc = "A Skinner Catering electric cooking range, Model VII-21, equipped with six burners and automatic turn-off. This one is well-used, and comes outfitted with a collection of pots, pans, skillets, and other dishes to cook any variety of food you can think of."
```

Comment:
Skinner Catering makes food, not food equipment. Ward-Takahashi would be a better choice here.

Reply:
Ya think so? I think W-T focuses mostly on electronics and mechanics, not simple stuff like home appliances. I might change the corp, but I think it'd probably be something smaller than W-T, but I also don't know our megacorp lore, so.